contrib/pylightning/lightning/lightning.py

```diff
@@ -804,17 +804,36 @@ def plugin_rescan(self):
         }
         return self.call("plugin", payload)
 
-    def sendpay(self, route, payment_hash, description=None, msatoshi=None):
+    def sendpay(self, route, payment_hash, *args, **kwargs):
         """
         Send along {route} in return for preimage of {payment_hash}
         """
-        payload = {
-            "route": route,
-            "payment_hash": payment_hash,
-            "description": description,
-            "msatoshi": msatoshi,
-        }
-        return self.call("sendpay", payload)
+
+        def _deprecated_sendpay(route, payment_hash, description=None, msatoshi=None):
```

Comment:
Giving `description` a default value seems strange here, after all we make sure we call it with a description below.

Reply:
Yes! I should remove this default.